All EIPIP Meetings/meeting 16.md

```diff
@@ -0,0 +1,399 @@
+
+# 5. Onboard EIP editors survey to set expectations of an EIP Editor
+
+Video | [38:40](https://youtu.be/UJmYpFkZGzY?t=2320)
+
+**Pooja** - right okay anymore anything more in this topic okay moving on the next item on the agenda is on boarding eip editor survey to set expectation of an eip editor i think edson 
+
+**Edson** - uh let me pull that up real quick um we only got three responses from eip editors i can go over the generally what they what they're saying um so the time commitment for new editors ie between is about five hours per week to zero hours per week um in that range the tasks that are expected to do uh make sure the eips are formatted their technically complete uh working with authors to get new eips into the repo looking at grammar structure the meaning uh if it's related to ethereum and uh looking at the auto merger bot if in cases it doesn't handle; good requirements for someone to become an eip editor is to already be helping um so anyone could review eaps editors are only needed to merge and someone that understands the difference between a standard and a good idea someone that understands the content that should go into standard and someone who's a good communicator and understands ethereum at least into an intermediate level and can handle contentious discourse and understands github
+
+**Pooja** - great so by this uh just one point i want to mention it again like by the task that has been suggested by the eip editors they want to focus it on uh if it is ethereum related um i don't know if it brings back to our uh conversation that we had sometimes back on like what would be the eip repository should be for maybe i'm not sure maybe we could have it a separate agenda item in one of the future meetings and then would like to discuss 
+
+**Micah** - it would be nice to settle that it's been ongoing when discussion not just between like my client and i for the one currently having but also like the other day someone put in something to the eips repo which was a specification for a new cryptographic primitive which um like is very general that's like something you would submit as an rfc to some other standardized body but they chose the ethereum repo is ethereum the ip's repo the right place for that like yes ethereum may use that cryptographic primitive at some point but there are definitely better places in my opinion to get feedback on like cryptography stuff than the ips repo so i think answering what is the scope of the ap's repo cover does it cover just ethereum and stuff that's going into ethereum and stuff that's specific to ethereum or does it cover a more broad scope including stuff that goes into that may or may not go into ethereum but is kind of related to the evm and things that run the evm and does it include like cryptography like which is tangentially related to ethereum so just answering that would be nice at some point it's not a super pressing matter though either
```

Comment:
it would be nice to settle that it's been ongoing when discussion not just between lightclient and I

Reply:
Do I really say the word "like" that much?  😢

Reply:
:laughing: your speech is more coherent than most @MicahZoltu